.github/workflows/mesheryctl-ci.yml

```diff
@@ -66,7 +66,7 @@ jobs:
           version: latest
           args: release --snapshot --skip-publish --rm-dist
   mesheryctl_docs:
-    if: "always() && github.event_name != 'pull_request' "
+    if: "always() && github.event_name != 'pull_request' && github.repository == 'meshery/meshery' "
```

Comment:
Thanks @alphaX86 @anshuhtwt for jumping in to review. 🎸 
I am ambiguous about this condition, could you please help me with this, whether I have correctly added the ```github.repository == 'meshery/meshery'``` condition or not.
Since, I searched on internet and got to know about the ```{{ ... }}``` thing, but couldn't find ```" ... "``` for the if block in GitHub Actions.